docs/user-guide/deployments.md

```diff
@@ -409,6 +409,12 @@ nginx-deployment-3066724191   0         0         1h
 
 Note: A paused Deployment cannot be scaled at this moment, and we will add this feature in 1.3 release, see [issue #20853](https://github.com/kubernetes/kubernetes/issues/20853). You cannot rollback a paused Deployment either, and you should resume a Deployment first before doing a rollback. 
 
+## Use Cases 
+
+### Canary Deployment
+
+If you want to run multiple versions of a container using the Deployment, you can just create multiple Deployments for different versions. 
```

Comment:
Can you expand on it a little bit more?  something like "create multiple deployments, and when you're happy with one, you can update your main deployment to the new version".  Maybe talk about using a Service over multiple deployments.  Just a little more for people who can't read your mind :)